test/test_util/mock_classes.dart

```diff
@@ -17,29 +17,6 @@ library mock_classes;
 import 'dart:async';
 import 'dart:html';
 
-// Tell dart2js that the `mockito` package only needs to reflect the specified mock/spied types.
-// This speeds up compilation and makes JS output much smaller.
-@MirrorsUsed(targets: const [
-  'dart.async.Timer',
-  'MockTimer',
-  // Also include Mock classes we use from w_test_tools.
-  'dart.dom.html.KeyEvent',
-  'dart.dom.html.HtmlDocument',
-  'w_test_tools.src.mock_classes.MockKeyEvent',
-  'w_test_tools.src.mock_classes.MockDocument',
-  'MockFileList',
-  'MockFile',
-  'MockFileUploadInputElement',
-  'dart.dom.html.FileList',
-  'dart.dom.html.File',
-  'dart.dom.html.FileUploadInputElement',
-  'MockSyntheticEvent',
-  'MockSyntheticMouseEvent',
-  'react.SyntheticEvent',
-  'react.SyntheticMouseEvent',
-], override: 'mockito')
-import 'dart:mirrors';
```

Comment:
@greglittlefield-wf were these needed? afaik, the only usage of mirrors within mockito was `spy()`, which I didn't find any usages of in `over_react`'s tests.

Reply:
Mm, looks like these were added under the false assumption that `Mock` classes used mirrors.

Away with them! 😄